### PR TITLE
Auto react in channels

### LINKED ISSUE #318: Discord autoreact
### Feature request

**Platform**
Meta discord

**Describe the feature**
Autoreact to all the messages in \#fiter-request and \#meta-feedback (and make a command to set it up maybe?)

**How the feature is useful**
Instead of staff or people having to react, Minehut Bot could do it and it would save time from staff and make the channels get no reaction spam.

---
**Use 👍 and 👎 reactions to indicate you agree or disagree with this suggestion! Feel free to discuss the idea in the comments.**

### PR DESCRIPTION
**Pull request by Sniped**
This adds configuration to auto react to any message posted in a configured channel. It was made for Minehut Meta channels #feedback-meta and #filter-requests.

Closes Minehut/Meta#318

**Configuration Changes**:
* Adds an `autoRole` property to the `features` property.

### CODE REVIEW

**Review by jellz — APPROVED** [review]

**Review by MiginsMC — COMMENTED** [review]
I feel like it would be cool if a user couldn't react with both reactions (maybe this could be something in the AutoReactChannel interface). I also feel like preventing users from reacting to their own posts could be something to consider (also possibly in the interface).

File: src/guild/config/guildConfigs.ts [hunk 1]
```diff
@@ -149,6 +149,18 @@ guildConfigs.set('546414872196415501', {
 			minimumChatPermission: PermissionLevel.Everyone,
 			overrides: [],
 		},
+		autoReact: {
+			channels: [
+				{
+					channel: '744075635114115142', // #meta-feedback
+					reactions: [':yes:546414872196415501', ':no:546414872196415501'],
```
Comment (theminecoder) [inline src/guild/config/guildConfigs.ts:156]: Has linking the emojis like this confirmed to be working? I can't remember the format off the top of my head but this looks odd

Comment (Sniped) [reply src/guild/config/guildConfigs.ts:156]: Yeah, I tested it on my end before making the PR. Learned something new when I tried it like this.

**Comment by Sniped** [comment]
> I feel like it would be cool if a user couldn't react with both reactions (maybe this could be something in the AutoReactChannel interface). I also feel like preventing users from reacting to their own posts could be something to consider (also possibly in the interface).

Yeah, I can probably push that out today.

**Review by quick007 — APPROVED** [review]

**Review by MiginsMC — APPROVED** [review]

**Review by theminecoder — APPROVED** [review]

**Pull request merged**